src/api-doc/api-doc.yaml

```diff
@@ -0,0 +1,2869 @@
+---
+openapi: 3.0.0
+info:
+  title: Elevate Mentoring
+  version: 1.0.0
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
```

Comment:
@ankitpws  this we need to remove  in all the api docs

Reply:
fixed